client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/CookieLoginDto.java

```diff
@@ -35,6 +35,10 @@
      */
     public String loginEndpointUrl;
 
+    /**
+     * Whether the [loginEndpointUrl] is a complete URL
+     */
+    public Boolean fullUrl;
```

Comment:
i don't think there is a need for such a boolean. we can simply analyze the content of `loginEndpointUrl` (eg, by parsing it in a URL object), and see if it is a full absolute URL, or a relative one (ie, no protocol/host/port info)